packages/opentelemetry-node/src/index.ts

```diff
@@ -15,3 +15,4 @@
  */
 
 export * from './NodeTracer';
```

Comment:
Do you think we should remove this now? As `NodeTracerFactory` is responsible for tracer creation and all.